components/event-publisher-proxy/pkg/metrics/collector.go

```diff
@@ -13,21 +13,11 @@ import (
 )
 
 const (
-	// BackendErrorsKey name of the backendErrors metric.
-	BackendErrorsKey = "eventing_epp_backend_errors_total"
-	// backendErrorsHelp help text for the backendErrors metric.
-	backendErrorsHelp = "The total number of backend errors while sending events to the messaging server"
-
 	// BackendLatencyKey name of the backendLatencyHelp metric.
 	BackendLatencyKey = "eventing_epp_backend_duration_milliseconds"
 	// backendLatencyHelp help text for the backendLatencyHelp metric.
 	backendLatencyHelp = "The duration of sending events to the messaging server in milliseconds"
 
-	// BackendRequestsKey name of the eventRequests metric.
-	BackendRequestsKey = "eventing_epp_backend_requests_total"
-	// backendRequestsHelp help text for event backendRequests metric.
-	backendRequestsHelp = "The total number of backend requests"
-
```

Comment:
We dont need these metrics? or do you think the info is already covered by the latency metric?

Reply:
the latency metric is a histogram.. it already provides you a count 
so figuring out the failed ones is just about filtering by the statusCode